apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/NamespaceController.java

```diff
@@ -242,11 +255,9 @@ public AppNamespace createAppNamespace(@PathVariable String appId,
   }
 
   /**
-   * env -> cluster -> cluster has not published namespace?
-   * Example:
-   * dev ->
-   *  default -> true   (default cluster has not published namespace)
-   *  customCluster -> false (customCluster cluster's all namespaces had published)
+   * env -> cluster -> cluster has not published namespace? Example: dev -> default -> true
```

Comment:
It's better to only format your own changes, or it would disrupt other's formating.